benchmark/buffers/buffer-write-string.js

```diff
@@ -3,36 +3,40 @@
 const common = require('../common.js');
 const bench = common.createBenchmark(main, {
   encoding: [
-    '', 'utf8', 'ascii', 'hex', 'UCS-2', 'utf16le', 'latin1', 'binary',
+    '', 'utf8', 'ascii', 'hex', 'utf16le', 'latin1',
   ],
   args: [ '', 'offset', 'offset+length' ],
-  len: [10, 2048],
-  n: [1e7]
+  len: [2048],
+  n: [1e6]
```

Comment:
`UCS-2` is identical to `utf16le` and `latin1` is identical to `binary`. The empty string should ideally also be removed since it's identical to `utf8`.

The mini buffer will be too fast to make any difference in any application and the iteration count was reduced to a number that is at least less far away from anything that would ever happen in a tight loop in a application.